docs/Install.md

```diff
@@ -35,11 +36,19 @@ Open the `config.json` file from the previous step with any text editor (such as
 
 **Token:**
 
-`"Token": "Your token here",` Replace `Your token here` with your Discord bot token. See [this article](https://www.writebots.com/discord-bot-token/) on instructions on how to obtain one.
+`"Token": "Token",` Replace `Token` with your Discord bot token. See [this article](https://www.writebots.com/discord-bot-token/) on instructions on how to obtain one. This is the backend login to the Discord application that users can invite to their Discord server.
 
 **OwnerID:**
 
-`"OwnerID": "ID",` Replace ID with your Discord ID. See [this article](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-#:~:text=On%20Android%20press%20and%20hold,name%20and%20select%20Copy%20ID.) on instructions on how to find this.
+`"OwnerID": "ID",` Replace ID with your Discord ID. See [this article](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-#:~:text=On%20Android%20press%20and%20hold,name%20and%20select%20Copy%20ID.) on instructions on how to find this. The account that is defined here will be able to send special maintenance commands as a direct message to the bot.
```

Comment:
Now that I look over this, we can shorten that link to remove everything after the #.